src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const robotCorrect = {};
+
+  for (let i = 0; i < robotValues.length; i++) {
```

Comment:
Here you can iterate not over your robotValues  array, but immediately over the robot array. And in a loop check if there is a field in the result object that is equal to robot[key] then return null, if not, then add the desired field with the desired value. This will shorten your code and allow you not to use the loop again